content/root/quick-start-guide.textile

```diff
@@ -14,6 +14,28 @@ When using Ably in a browser, simply include the Ably Javascript library within
 
 bc[html]. <script src="//cdn.ably.io/lib/ably.min.js"></script>
 
+blang[objc].
+  Add this line to your application's Podfile:
+
+  bc. pod 'Ably', '~> 0.8'
+
+  And install with @pod install@. Then in your files: 
+
+  ```[objc]
+  #import "Ably.h"
+  ```
+
+blang[swift].
+  Add this line to your application's Podfile:
+
+  bc. pod 'Ably', '~> 0.8'
+
+  And install with @pod install@. Then in your files: 
+
+  ```[objc]
```

Comment:
> isn't this supposed to swift?

Fixed at 2661702.

> you'll see that you need to perhaps change that copy to be included in all languages other than the ones you've suggested

I don't know how this should be done for all languages. I realize that the browser copy should be only for JavaScript, but shouldn't that be in a different pull request?